### PR TITLE
Coachella concert capstone project

### PR DESCRIPTION
**Pull request by codewanderlust**
## In this pull request I did the following ;
- Added the home page and the about page using a mobile-first approach
-  Added the HTML code for the Featured Artist section, dynamically with the help of Javascript
- Used a media query to adjust the elements of the page at different screen sizes
- Customized the Readme and added a screenshot and a walkthrough  video of the project

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
```
Comment (levy002) [inline about.html:39]: - [ ] Kindly make sure to add the top navigation bar which is shown on the provided design. Please make sure to add it in order to meet the provided design :wink: 

   - ### YOUR DESIGN
    image

   - ### PROVIDED FIGMA
    image

File: README.md [hunk 2]
```diff
@@ -132,154 +104,44 @@ Example command:
...
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+- I would like to thank Microverse for providing me with the materials that made it possible to finish this exercise
```
Comment (levy002) [inline README.md]: - [ ] Kindly make sure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.css [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+}
+
+.footer {
+  display: grid;
+  grid-template-columns: 1fr 1.5fr;
+  align-items: center;
+  justify-content: center;
+  gap: 2.4rem;
+  padding: 1.2rem;
+}
```
Comment (levy002) [inline about.css]: - [ ] Kindly make sure your footer background in `about` page is dark as shown in the provided design. :wink: 

    - ### Your design
    image

    - ### Provided design
    image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,732 @@
...
+.program-details {
+  display: flex;
+  flex-direction: row;
+  gap: 1.8rem;
+  align-items: center;
+  justify-content: center;
+  background-color: #e6e6e625;
+  margin: 1.4rem;
+  padding: 1.2rem;
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure to align the card's content correctly in order to meet the provided design. :wink: 

    - ### YOUR DESIGN
    image

    - ### PROVIDED FIGMA DESIGN
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @RoyJumah   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly set up the linter for `es-lint` so it can run to check the javascript code. Kindly go ahead and use [this setup.](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.css [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+
+  /* filter: grayscale(50%); */
+}
+
+.footer--about {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  column-gap: 2.4rem;
+}
```
Comment (Olamarx) [inline about.css]: - [ ] Kindly make the footer to be centered as the design you followed.

### Your design
image

### Behance
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @RoyJumah, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Linters passed ✔️ 
- Readme file update 👍🏼 
- Functionalities ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @RoyJumah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linters passed ✔️ 
- Functionalities ✅ 
- Home and About pages ✔️ 
- Project Deployment ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**